File: Medium-Questions/08-Ascii-input/Python/eight.py
import sys

# get input from user
tmp = input("Enter character: ")

# get ascii value of character
tmp = ord(tmp)

# if tmp between 48 to 57 so user input is number
if (tmp >= 48 and tmp <= 57):
    print("number")
    sys.exit(0)

# if tmp between 65 to 90 so user input is number
elif(tmp >= 65 and tmp <= 90):
    print("upper case alphbet")
    sys.exit(0)
    
# if tmp between 97 to 122 so user input is number
elif(tmp >= 97 and tmp <= 122):
    print("Lower case alphabet")        
    sys.exit(0)
    
else:
    print("its a symbol")
    sys.exit(0)

    # ascii table 
# https://www.asciitable.com/asciifull.gif 

File: Medium-Questions/08-Ascii-input/C#/eight.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ascii
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Enter a single character");
            string temp = Console.ReadLine();

            if (temp.Length > 1)
            {
                Console.WriteLine("Input is not a Single Character");
                Console.ReadKey();
                return;
            }


            byte[] ASCIIvalues = Encoding.ASCII.GetBytes(temp);
            foreach (var c in ASCIIvalues)
            {
                if (c >= 48 && c <= 57)
                {
                    Console.WriteLine("Number :) ");
                }
                else if (c >= 65 && c <= 97)
                {
                    Console.WriteLine("Uper case Alphabet :) ");
                }
                else if (c >= 97 && c <= 122)
                {
                    Console.WriteLine("Lower case Alphabet :) ");
                }
                else
                {
                    Console.WriteLine("its a symbol");
                }

            }

            Console.ReadKey();
        }
    }
}
